cognite/neat/_session/_show.py

```diff
@@ -14,26 +18,147 @@ class ShowAPI:
     def __init__(self, state: SessionState) -> None:
         self._state = state
         self.data_model = ShowDataModelAPI(self._state)
+        self.instances = ShowInstanceAPI(self._state)
+
+
+class ShowInstanceAPI:
+    def __init__(self, state: SessionState) -> None:
+        self._state = state
+
+    def __call__(self) -> Any:
+        if not self._state.store.graph:
+            print("No graph data available.")
+            return None
```

Comment:
Suggest just assuming the graph is available and raise an error with hint on how to load data if it is not there.